src/app/models/vehicle-tech-record.model.ts

```diff
@@ -138,12 +137,49 @@ export enum VehicleSubclass {
   W = 'w'
 }
 
```

Comment:
Need to go in here and set tech record to use a single object opposed to array as well